mava/environment_loop.py

```diff
@@ -570,7 +574,7 @@ def run_episode_and_log(self) -> loggers.LoggingData:
         self._logger.write(results)
         return results
 
-    def run(
+    def run(  # noqa: C901
```

Comment:
c901 is for "too complex", can we not simplify? Might be the complicated return statement in `should_terminate` function? Not sure really.

Reply:
I removed this statement in the environment loop PR since we don't need the should terminate any more `https://github.com/instadeepai/Mava/pull/773/files`